Neos.ContentGraph.DoctrineDbalAdapter/Classes/Domain/Projection/GraphProjector.php

```diff
@@ -55,6 +57,19 @@ class GraphProjector implements ProjectorInterface
      */
     protected $client;
 
+    /**
+     * @Flow\Inject
+     * @var VariableFrontend
+     */
+    protected $processedEventsCache;
+
+    public function hasProcessed(NodeCommandResult $commandResult): bool
+    {
+        foreach ($commandResult->getPublishedEvents() as $event) {
+
```

Comment:
At this point we have an `DomainEventInterface` but we'd need the `RawEvent` (or at least the event identifier)